schema/rpc.fbs

```diff
@@ -521,3 +521,22 @@ table SetPauseTrackingRequest {
 
 /// Clears mounting reset data, defaulting to the manually set mounting orientations
 table ClearMountingResetRequest {}
+
+/// Common folders often used in computers for storing files
+enum ComputerDirectory: uint8 {
+    DOCUMENTS,
+}
+
+/// Used for the server to save a file and have it prompt in the user side
+table SaveFileNotification {
+    /// Binary data of the file
+    data: [uint8] (required);
+    /// MIME type of file if one exists, use `file_extension` otherwise
+    mime_type: string;
+    /// Use MIME type preferably if one exists
+    file_extension: string;
+    /// Directory recommended to save the file on
+    expected_dir: ComputerDirectory;
```

Comment:
Make this enum optional, since its a scalar I believe (fact check me pls) that it defaults to the first (0th) element